Enterprise-CMCS/managed-care-review
# MCR-4329 & MCR-4330: inline error, alert, checkbox, and radio tagging

## Summary
[MCR-4329](https://jiraent.cms.gov/browse/MCR-4329)
[MCR-4330](https://jiraent.cms.gov/browse/MCR-4330)

- Add logging for only Alert banners from errors or validations.
   - `DocumentWarningBanner`
   - `GenericApiBanner`
   - `UserAccountWarningBanner`
   - `ErrorAlert`
   - `ErrorAlertFailedRequest`
   - `UploadErrorAlert`
- Add logging for inline error messages
   - Implement logging in `PoliteErrorMessage`
   - All inputs in the state submission form
   - QA uploads
- Add checkbox logging to `FieldCheckBox` component
- Add radio logging to `FieldRadio` component

#### Related issues

#### Screenshots

#### Test cases covered



## QA guidance

- If you run locally, you will see Tealium log into the console twice because `React.StrictMode` will render things twice and cause `useEffect` to fire twice. This does not happen in the other environments.
